### ISSUE TITLE
Make a simple web app that generates a copy-pastable embed

### ISSUE THREAD

**Issue by osdiab** [everydotorg/donate-button#127]
Once #124 is in, then we can make a preact component that asks for info about the nonprofit, and spits out a copy-pastable embed, which we can ultimately host from embeds.every.org and also potentially the main Every.org site.

Would be nice if this was exported to `npm` for our main app to consume. eventually we can also just use API calls to pull most of the requisite data from our API, using the nonprofit slug (and just allow things to be overridden by hadn in this component); it might already work as is but need to check that.

**Comment by osdiab** [comment]
cc @markulrich @rahulgi @wednesdave

**Comment by martinbianchi** [comment]
@osdiab I have some doubts about this one. 
Should we create this on a new package inside this repo? Or create a new component and export it as we did with #124 ?
Also, about the information that we will ask to generate the embed, we want to get from the non-profit all the information needed to configure the split-layout widget? I'm thinking that maybe we will need some designs if that's the idea

**Comment by osdiab** [comment]
Yeah, we should definitely design this and also add more helper functionality to auto populate as much of this information as possible from a nonprofit slug. I’ll make some time to spec this out further before we dive in.
